admin/class-admin.php

```diff
@@ -234,7 +234,7 @@ public function do_menu_changes() {
 			array(
 				'wi-volunteer-management',
 				'',
-				__( 'Help & Settings', 'wired-impact-volunteer-management' ),
+				__( apply_filters( 'wivm_help_and_settings_menu_item_name', 'Help & Settings' ), 'wired-impact-volunteer-management' ),
```

Comment:
Since there's already a filter on line 253 called `wivm_submenu_pages` can we get rid of this one and just use that one?

Reply:
Great call, that pre-existing hook allows us to modify the submenu page name for each of the submenu pages which is really nice! I'll get this filter removed.